app/views/posts/index.html.erb

```diff
@@ -87,8 +87,20 @@
             </footer>
           <% end %>
 
-          <div class="comment-box p-2">
```

Comment:
Does this need to be deleted? This div is a really crucial element in the Javascript for adding comments 😅